src/decision/test/lidar-decision-test.cpp

```diff
@@ -7,6 +7,8 @@
 #include <LidarDecision.h>
 #include <gtest/gtest.h>
 
+#define distance 1.0f
```

Comment:
generally speaking, in C++ code, we want to avoid defining constants like this. In this case, since you're only using it in `LidarObstacle` tests, I would suggest putting it in the `LidarObstacleTest` class declaration, **declaring** it where other variables like `scan1` are declared, and **defining** it in the `SetUp` function, similar to how `scan1` is declared.